tensorflow/tensorboard/components/tf-dashboard-common/dashboard-style.html

```diff
@@ -54,6 +54,14 @@
         display: block;
       }
 
+      .log-option-button {
```

Comment:
tf-event-dashboard doesn't currently have CSS rules of its own, all css rules are being dumped in dashboard-style.